acceptance/setup/aio/pre-suite/015_PackageHostsPresets.rb

```diff
@@ -1 +1,9 @@
-master['use-service'] = true
+if options['is_puppetserver']
+  master['use-service'] = true
+  master['puppetservice'] = 'puppetserver'
+  master['puppetserver-confdir'] = '/etc/puppetlabs/puppetserver/conf.d'
```

Comment:
We could put them in the global aio options hash, but we would then need to override them in the global CONFIG section of each passenger specific host files. (We have to create new passenger host files as decribed below).

Another alternative would be to create separate aio and passenger options files. But since we already require passenger specific host files, I thought it would be better to identify the master host as "passenger" in the host file. Then all of the logic for how to configure passenger vs puppetserver is in one place, in this setup step.